File: Laminatoria/DTO/OrderRequest.cs
using Laminatoria.Infrastructure;
using Laminatoria.Models;
using Microsoft.EntityFrameworkCore.Metadata.Internal;
using System.ComponentModel.DataAnnotations.Schema;

namespace Laminatoria.DTO
{
    public class OrderRequest
    {
        public int Id { get; set; }
        public Contact Contacts { get; set; }
        public string Address { get; set; }
        public Status Status { get; set; }
        public string Comments { get; set; }
        public DateTime Date { get; set; }
        public string Delivery { get; set; }
        public decimal Summary { get; set; }
        public List<OrderItemRequest> OrderItems { get; set; } = new List<OrderItemRequest>();
    }
}

File: laminatoria-client/src/app/classes/orderRequest.ts
import { Contacts } from './contacts'
import { OrderItemRequest } from './orderItemRequest'

export class OrderRequest {
	id: number
	contacts: Contacts
	status: number
	address: string
	comments: string
	date: Date
	orderItems: OrderItemRequest[]
	delivery: string
	summary: number

	constructor(
		id: number,
		contacts: Contacts,
		status: number,
		address: string,
		comments: string,
		date: Date,
		orderItems: OrderItemRequest[],
		delivery: string,
		summary: number
	) {
		this.id = id
		this.contacts = contacts
		this.status = status
		this.address = address
		this.comments = comments
		this.date = date
		this.orderItems = orderItems
		this.delivery = delivery
		this.summary = summary
	}
}
